code/03_census_and_bed_days.R

```diff
@@ -0,0 +1,868 @@
+# 
+# datafile7<-datafile7 %>% mutate(notgpled,
+#                                 if_else(age_grp=="All" & 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+# 
+# datafile7<-datafile7 %>% mutate(acute,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,acute)))
+# 
+# datafile7<-datafile7 %>% mutate(gpled,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+# 
+# 
+# datafile7<-datafile7 %>% mutate(notgpled,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+
+#Note: used datafile6 as the creation of datafile7 doesn't add anything new - PMc: check .
+
+### 13. Save out census file ----
+
+write_sav(datafile6,paste0(filepath,"All census data.sav"))
+
+write.xlsx(datafile6,paste0(filepath,"All census data.xlsx"))
+
+table(datafile6$areaname)
+
+
+
+
+
+
+################################################################################
+### 14. bed days data ----
+################################################################################
+
+#Get filespath+main bed days file.SAV
+
+#datafile11<-read_spss(paste0(filepath,"main bed days file.sav"))
+
+
+datafile11<-datafile %>% mutate(Healthboard=substring(Healthboard, 5)) # change nhs_board and remove the 'NHS_'
+
+hbbeddaysdiffage_grp<- datafile11 %>% 
+  group_by(Healthboard,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+
+#write_sav(hbbeddaysdiffage_grp,paste0(filepath,"hb bed days diff age_grp_R.sav")) # save out file
+
+labeddaysdiffage_grp<- datafile11 %>% 
+  group_by(LocalAuthorityArea,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+
+#write_sav(labeddaysdiffage_grp,paste0(filepath,"la bed days diff age_grp_R.sav")) # save out file
+
+datafile12<-datafile11 %>% mutate(Healthboard="Scotland")
+
+Scotbeddaysdiffage_grp<- datafile12 %>% 
+  group_by(Healthboard,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+
+#write_sav(Scotbeddaysdiffage_grp,paste0(filepath,"Scot bed days diff age_grp_R.sav")) # save out file
+
+
+# add files
+
+Scotlandhbbeddays<-rbind(Scotbeddaysdiffage_grp,hbbeddaysdiffage_grp)
```

Comment:
See "Style Guide" section in general feedback. 
You use `bind_rows()` throughout but move to `rbind()` here. `bind_rows()` as a `dplyr` function should replace `rbind()` here.
```suggestion
Scotlandhbbeddays <- bind_rows(Scotbeddaysdiffage_grp, hbbeddaysdiffage_grp)
```